docs/DeveloperGuide.md

```diff
@@ -151,37 +126,80 @@ Through Patient Manager, general practitioners are able to manage patients faste
 |v2.0|GP in a polyclinic|know if I entered a invalid Patient ID|make sure no mistake is made recording the patient's ID|
 |v2.0|GP in a polyclinic|load and save existing data|work on the data on another device|
 
-## Non-Functional Requirements
+## Appendix C: Non-Functional Requirements
 
 1. Should work on any mainstream OS as long as it has Java 11 or above installed. 
 2. Should be able to hold up to 1000 patients without a noticeable sluggishness in performance for typical usage.
 3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should 
-   be able to accomplish most of the tasks faster using commands than using the mouse. 
-   
-{more to be added}
-
-## Glossary
-
-* *glossary item* - Definition
-
-## Testing
-
-### Automated Testing
-We have both JUnit Test and IO Redirection Test. To run these tests, execute these commands in a shell or CMD:
-```
-### For JUnit tests
-# *nix OS and MacOSX bash
-./gradlew check
-# Windows CMD
-gradlew.bat check
-
-### For I/O redirection tests
-cd text-ui-test
-# *nix OS and MacOSX bash
-./runtest.sh
-# Windows CMD
-runtest.bat
-```
-
-### Manual Testing
-The current version does not support storing the data on the local drive. To start with the manual testing process, you may refer to the "QuickStart" section of the [User Guide](UserGuide.md).
+   be able to accomplish most of the tasks faster using commands than using the mouse.
+4. The data should be stored locally and should be in a human editable text file.
+5. The application should work without requiring an installer.
+6. The application should be at most 100 MB in size.
+7. The application should not rely on any remote server, or database management system.
```

Comment:
As per SE-EDU's markdown guide, using all `1.` might be better.

Reply:
Done!